app/views/relationships/create.js.erb

```diff
@@ -0,0 +1,2 @@
+$("#follow_form").html("<%= escape_javascript(render('users/unfollow')) %>");
+$("#followers").html('<%= @user.followers.size %>');
```

Comment:
dúng nháy đôi đơn lộn xộn quá em

Trong ruby toàn bộ dùng nháy đôi
trong file js thì dùng nháy đơn nhé

Reply:
đã comment mà ko chú ý nhé.. (tat)